README.md

```diff
@@ -81,6 +81,23 @@ read(url, {
   });
 ```
 
+- `candidatesFilters` which allow set your own filters for candidate tags.
+
+options.candidatesFilters = [callback(obj, index)]
+```javascript
+read(url, {
+  candidatesFilters: [
+    function (obj) {
```

Comment:
`obj` doesn't describe what this function is passed. Maybe something like `candidateNode`?